src/coreclr/jit/CMakeLists.txt

```diff
@@ -92,6 +92,8 @@ function(create_standalone_jit)
     target_include_directories(${TARGETDETAILS_TARGET} PRIVATE "../tools/Common/JitInterface")
 
     # Only look at "our" LLVM with LLVM_CMAKE_CONFIG.
+    set(LLVM_DIR $ENV{LLVM_CMAKE_CONFIG})
+    set(CMAKE_CXX_STANDARD 17) 
```

Comment:
Does the previous approach not work? It's fine if it doesn't, but then it needs to be deleted and this line also converted to be target-based:
```
set_target_properties(${TARGETDETAILS_TARGET} PROPERTIES CXX_STANDARD 17)
```

Reply:
reverted